hadoop-hdds/docs/content/feature/PrefixFSO.md

```diff
@@ -62,32 +63,17 @@ Following picture describes the OM metadata changes while performing a rename
 {{< image src="PrefixFSO-Rename.png">}}
 
 ## Configuration
-By default the feature is disabled. It can be enabled with the following
- settings in `ozone-site.xml`:
-
-```XML
-<property>
-   <name>ozone.om.enable.filesystem.paths</name>
-   <value>true</value>
-</property>
-<property>
-   <name>ozone.om.metadata.layout</name>
-   <value>PREFIX</value>
-</property>
-```
 
-In reference to efforts towards supporting protocol aware buckets 
-within an Ozone cluster, the following configuration can be used 
-to define the default value for bucket layout during bucket creation 
-if the client has not specified the bucket layout argument. 
+The following configuration can be used to define the default value for bucket layout during bucket creation
```

Comment:
@JyotinderSingh can you include `ozone-site.xml`. Apart from this doc looks pretty good!

The following configuration can be configured in `ozone-site.xml` to define the default value for bucket layout during bucket creation if the client has not specified the bucket layout argument.

Reply:
I have added this in the latest patch, thank you for the review!